test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -1,4 +1,4 @@
-import { RequestLogger, RequestMock, Selector } from 'testcafe';
+import { RequestLogger, RequestMock } from 'testcafe';
```

Comment:
Why does this PR merge directly to master instead of 3.7? This fix is on 3.7 I believe.

Reply:
this is more refactor work and doesn't worth to go with patch.
we can deal with merge conflict (if any) when back port patch changes to master.

Reply:
I think this is just test clean up which does not need to be patched immediately by putting it to `3.7`. PRs merged to master will go out at the end of the month when we release a minor version.

Reply:
Make sense. Thanks for the clarification!